.chloggen/lb-exporter-dummy-endpoint.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: loadbalancingexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: A change to the OTLP exporter causes the load-balancing exporter to fail during startup as no endpoints are defined at that stage.
```

I'm not sure if this description will be understandable to users. How about something like this?
```suggestion
note: Don't require `endpoint` in `otlp` configuration
```

---

How about:

> Fixes a bug where the endpoint become required, despite not being used by the load balancing exporter.